bank/html-podstrony/bilety/zakupione-bilety.html

```diff
@@ -10,73 +10,113 @@
             <span id="pop-up-text">Lorem Ipsum...</span>
         </div>
     </div>
-    <div id="ticket-information" class="homepage-section">
-        <div class="ticket-information-element">
-            <div class="left-column">
-                <div class="ticket-left-column-element">Park & Ride Kraków</div>
-            </div>
-            <div class="right-column">
-                <div class="first-row">Ważny do: 30-06-2024</div>
-                <div class="second-row">
-                    <div id="details" class="second-row-element">Szczegóły</div>
-                    <div class="second-row-element">Drukuj</div>
-                </div>
-            </div>
-        </div>
 
-        <div class="ticket-information-element">
-            <div class="ticket-placeholder">
-                <div class="left-column">
-                    <div class="ticket-left-column-element"></div>
-                </div>
-                <div class="right-column">
-                    <div class="first-row"></div>
-                    <div class="second-row">
-                        <div class="second-row-element"></div>
-                        <div class="second-row-element"></div>
-                    </div>
-                </div>
-            </div>
-        </div>
+    <div id="ticket-information" class="homepage-section">
```

Comment:
![image](https://github.com/Elyrwag/ig-strona-bankowa/assets/147379179/fc21b6fc-b4b6-447a-a587-a3e686d38907)

co wy na to, aby ten tekst wyśrodkować względem środka strony?

Reply:
Według mnie jest okej tak jak jest, ewentualnie można by Szczegóły i Drukuj do prawej dać